Source/santad/SNTSyncdQueue.mm

```diff
@@ -15,23 +15,28 @@
 
 #import "Source/santad/SNTSyncdQueue.h"
 
+#include <memory>
+
 #import "Source/common/MOLXPCConnection.h"
 #import "Source/common/SNTLogging.h"
 #import "Source/common/SNTStoredEvent.h"
 #import "Source/common/SNTXPCSyncServiceInterface.h"
+#include "Source/common/SantaCache.h"
+#include "Source/common/String.h"
 
 @interface SNTSyncdQueue ()
-@property NSCache<NSString *, NSDate *> *uploadBackoff;
 @property dispatch_queue_t syncdQueue;
 @end
 
-@implementation SNTSyncdQueue
+@implementation SNTSyncdQueue {
+  // TODO: Eventually replace with an LRU.
+  std::unique_ptr<SantaCache<std::string, NSDate *>> _uploadBackoff;
+}
 
 - (instancetype)init {
   self = [super init];
   if (self) {
-    _uploadBackoff = [[NSCache alloc] init];
-    _uploadBackoff.countLimit = 128;
+    _uploadBackoff = std::make_unique<SantaCache<std::string, NSDate *>>(128);
```

Comment:
I think we should consider bumping to 256 here. Seems maybe high since this cache should only be filled with data from true blocks. But given the object sizes are small, and given the non-LRU semantics it might be worth it.

Reply:
SGTM